shepherd-model/src/main/java/com/devsda/platform/shepherd/model/DataSourceDetails.java

```diff
@@ -0,0 +1,100 @@
+package com.devsda.platform.shepherd.model;
+
+import com.devsda.utils.httputils.constants.Protocol;
+
+import java.util.ArrayList;
+
+public class DataSourceDetails {
+
+    private String prefix;
+    private String user;
+    private String password;
+
+    private ArrayList<String> clusters;
+
+    private String path;
+    private String params;
+
+    public String getDb_name() {
+        return db_name;
+    }
+
+    public void setDb_name(String db_name) {
```

Comment:
Don't use Snake case convention. Let it be there in Python.:-D. Don't ruin the beauty of Java. :-)